src/panels/config/automation/trigger/types/ha-automation-trigger-calendar.ts

```diff
@@ -46,7 +46,7 @@ export class HaCalendarTrigger extends LitElement implements TriggerElement {
             ],
           ],
         },
-        { name: "offset", selector: { duration: {} } },
+        { name: "offset", required: true, selector: { duration: {} } },
```

Comment:
Perhaps it makes sense for this to be clearable but I agree that we should preserve the current logic in this PR